services/core-api/tests/mines/documents/test_mine_document_list_resource.py

```diff
@@ -95,7 +106,13 @@ def test_list_resource_filter_by_project_summary_guid(self, test_client, db_sess
         get_data = json.loads(get_resp.data.decode())
 
         assert len(get_data['records']) == 2
-        assert str(ps.documents[0].mine_document_guid) == get_data['records'][0]['mine_document_guid']
+
+        guids = (str(ps.documents[0].mine_document_guid), str(ps.documents[1].mine_document_guid))
+
+
+        assert get_data['records'][0]['mine_document_guid'] in guids
+        assert get_data['records'][1]['mine_document_guid'] in guids
```

Comment:
Hopefully this makes this test a bit less flaky, think the documents are just not returned in any particular order, so sometimes it just fails